script.js

```diff
@@ -17,3 +20,165 @@ mobNavItems.forEach((mobNavItem) => {
+        <h3>${projectInfo[index][1]}</h3>
+        <img src="./design/assets/Counter.svg" alt="Decorative separator dot image">
+        <h3>${projectInfo[index][2]}</h3> 
+      </div>
+      <img class="modal-desktop-img" src="${desktopImage[index]}" />
+      <div class="modal-details">
+        <p>${projectDescriptions}</p>
+        <div>
+        <ul class="project-used-techs modal-used-techs">
+          <li>${projectSkills[index][0]}</li>
+          <li>${projectSkills[index][1]}</li>
+          <li>${projectSkills[index][2]}</li>
+        </ul>
+        <div class="see-more-btns">
+          <a onclick="window.open('https://z4cope.github.io/Mobile-portfolio/')" target="_blank" class="call-to-action-btns modal-btn">See live <img src="./design/assets/see-more.svg" /></a>
+          <a onclick="window.open('https://github.com/z4cope/Mobile-portfolio')" target="_blank" class="call-to-action-btns modal-btn">See Source <img src="./design/assets/github-link.svg" /></a>
+        </div>
+        </div>
+      </div>
+      `;
+      const closeBtn = document.getElementById('close-modal');
+      closeBtn.addEventListener('click', () => {
+        modalContainer.style.transform = 'translateX(-100%)';
+        document.body.classList.remove('stop-scrolling');
+      });
+    });
+  });
+});
+
+const userForm = document.getElementsByTagName('form')[0];
+const userEmail = document.getElementById('user-email');
+let validValue = false;
+const messageContainer = document.getElementById('message-box');
+let validEmailValue;
+
+userEmail.addEventListener('input', (e) => {
+  validEmailValue = e.target.value.toLowerCase();
+  if (validEmailValue === e.target.value) {
+    validValue = true;
+  } else {
+    validValue = false;
+  }
+});
+
+userForm.addEventListener('submit', (e) => {
+  if (!validValue) {
+    const message = document.createElement('p');
+    message.innerText = 'Please fill your email properly in lower case';
+    messageContainer.append(message);
+    messageContainer.style.opacity = '1';
+    e.preventDefault();
+  }
+});
+
+const userFormData = {
+  name: '',
+  email: '',
+  message: '',
+};
+
+userForm.elements.inputName.addEventListener('blur', () => {
+  userFormData.name = userForm.elements.inputName.value;
+});
+
+userForm.elements.inputEmail.addEventListener('blur', () => {
+  userFormData.email = userForm.elements.inputEmail.value;
+});
+
+userForm.elements.inputMessage.addEventListener('blur', () => {
+  userFormData.message = userForm.elements.inputMessage.value;
+  window.localStorage.setItem('userFormData', JSON.stringify(userFormData));
+});
+
+userForm.elements.inputName.value = JSON.parse(
+  window.localStorage.getItem('userFormData'),
+).name;
+
+userForm.elements.inputEmail.value = JSON.parse(
+  window.localStorage.getItem('userFormData'),
+).email;
+
+userForm.elements.inputMessage.value = JSON.parse(
+  window.localStorage.getItem('userFormData'),
+).message;
```

Comment:
- According to the project requirements, the form details should be updated in the local storage on user input. But I noticed this isn't working properly in your project. The local storage only gets updated when I click outside the input elements. I'd suggest you add an 'input' event listener to your form. Please try to fix this 👍 👍 

Your project(Pay attention to the data in the inputs and what was preserved to the local storage) 👇:
<img width="944" alt="localstorage" src="https://user-images.githubusercontent.com/73607512/195600330-de113aa8-bd41-474f-8fae-51e41b1b6ff3.PNG">